gatorgrouper/utils/group_scoring.py

```diff
@@ -11,8 +11,8 @@ def score_group(student_identifers):
     return score
 
 
-def score_groups(student_groups):
-    """ score multiple groups """
+def calculate_ave(student_groups):
```

Comment:
This should probably use the abbreviation `avg` instead of `ave`.

Reply:
I have changed the abbreviation from ave to avg in all necessary locations.